lib/capistrano/tasks/newrelic.rake

```diff
@@ -18,7 +17,7 @@ namespace :newrelic do
         new_relic_api_key = fetch(:new_relic_api_key) || ENV['NEW_RELIC_API_KEY']
         fail ":new_relic_api_key must be set - \"set :new_relic_api_key, 'yourkey'\"" unless new_relic_api_key
         response = Faraday.post do |req|
-          req.url fetch(:new_relic_url)
+          req.url fetch(:new_relic_url) || "https://api.newrelic.com/v2/applications/#{ENV['NEW_RELIC_APP_ID']}/deployments.json"
```

Comment:
Just a clarifying question, how is `NEW_RELIC_APP_ID` getting passed in as an environment variable?